src/components/item/sharing/CsvInputParser.tsx

```diff
@@ -1,53 +1,56 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import { Record } from 'immutable';
-import { Button, Loader } from '@graasp/ui';
-import Grid from '@material-ui/core/Grid';
-import Box from '@material-ui/core/Box';
-import Alert from '@material-ui/lab/Alert';
-import AlertTitle from '@material-ui/lab/AlertTitle';
-import DialogContent from '@material-ui/core/DialogContent';
-import DialogActions from '@material-ui/core/DialogActions';
-import buildI18n from '@graasp/translations';
-import PublishIcon from '@material-ui/icons/Publish';
-import DialogTitle from '@material-ui/core/DialogTitle';
-import Dialog from '@material-ui/core/Dialog';
-import DialogContentText from '@material-ui/core/DialogContentText';
-import { useTranslation } from 'react-i18next';
-import { makeStyles } from '@material-ui/core';
+import { RecordOf } from 'immutable';
 import Papa from 'papaparse';
+
+import PublishIcon from '@mui/icons-material/Publish';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import Box from '@mui/material/Box';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Grid from '@mui/material/Grid';
+
+import { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
 import { MUTATION_KEYS } from '@graasp/query-client';
+import { Item } from '@graasp/sdk';
+import { BUILDER, COMMON, namespaces } from '@graasp/translations';
+import { Button, Loader } from '@graasp/ui';
+
+import { useBuilderTranslation } from '../../../config/i18n';
 import { useMutation } from '../../../config/queryClient';
-import { PERMISSION_LEVELS } from '../../../enums';
 import {
   SHARE_ITEM_CSV_PARSER_BUTTON_ID,
   SHARE_ITEM_CSV_PARSER_INPUT_BUTTON_ID,
   SHARE_ITEM_FROM_CSV_ALERT_ERROR_ID,
   SHARE_ITEM_FROM_CSV_RESULT_FAILURES_ID,
 } from '../../../config/selectors';
+import { PERMISSION_LEVELS } from '../../../enums';
 
+const label = 'shareItemFromCsvLabel';
 const allowedExtensions = ['.csv'].join(',');
 
-const useStyles = makeStyles(() => ({
-  buttonWrapper: {
-    textAlign: 'center',
-  },
-}));
+type Props = {
+  item: RecordOf<Item>;
+};
 
-const CsvInputParser = ({ item }) => {
-  const classes = useStyles();
-  const { t } = useTranslation();
+const CsvInputParser: FC<Props> = ({ item }) => {
+  const { t } = useBuilderTranslation();
+  const { t: messageT } = useTranslation(namespaces.messages);
+  const { t: commonT } = useTranslation(namespaces.common);
```

Comment:
Read other messages about translations. An idea:

```
  const { t: translateBuilder } = useBuilderTranslation();
  const { t: translateMessages } = useMessagesTranslation();
  const { t: translateCommon } = useCommonTranslation();
```

In any case, at least change to `messagesT` for consistency.